designer/client/src/components/toolbarComponents/Panel.tsx

```diff
@@ -0,0 +1,49 @@
+import { Collapse, styled } from "@mui/material";
+import { getContrastColor, getDarkenContrastColor } from "../../containers/theme";
+
+const __panelBorder = "--panel-Border";
+const __panelHeaderText = "--panel-HeaderText";
+const __panelColor = "--panel-Color";
+const __panelText = "--panel-Text";
+
+export const PanelHeader = styled("div")(({ theme }) => ({
```

Comment:
There is a difference in the styles of the toolbar header text on the Safari browser (tested on Safari 16.4). On Brave, it works as expected.

After:

<img width="289" alt="image" src="https://github.com/TouK/nussknacker/assets/9945753/6451419e-54fb-463d-b7f6-f8a90d62ff97">

Before:

<img width="290" alt="image" src="https://github.com/TouK/nussknacker/assets/9945753/cb3afd6a-f7c2-4f68-9fba-177f3f0f8171">